R/runDPCoA.R

```diff
@@ -4,58 +4,41 @@
 #' \code{ade4} package in typical fashion. Results are stored in the
 #' \code{reducedDims} and are available for all the expected functions.
 #'
-#' @param x For \code{getDPCoA}, a numeric matrix of expression values
-#'   where rows are features and columns are cells.
-#'   Alternatively, a \code{TreeSummarizedExperiment} containing such a matrix.
+#' @inheritParams calculateDMN
 #'
+#' @details
 #'   For \code{addDPCoA} a \linkS4class{TreeSummarizedExperiment} containing the
 #'   expression values as well as a \code{rowTree} to calculate \code{y} using
 #'   \code{\link[ape:cophenetic.phylo]{cophenetic.phylo}}.
 #'
 #' @param y a \code{dist} or a symmetric \code{matrix} compatible with
 #'   \code{ade4:dpcoa}
 #'
-#' @param ncomponents Numeric scalar indicating the number of DPCoA dimensions
-#'   to obtain.
+#' @param ncomponents \code{Numeric scalar}. Indicates the number of DPCoA dimensions
+#'   to obtain. (Default: \code{2})
 #'
-#' @param ntop Numeric scalar specifying the number of features with the highest
+#' @param ntop \code{Numeric scalar}. Specifies the number of features with the highest
 #'   variances to use for dimensionality reduction. Alternatively \code{NULL},
-#'   if all features should be used. (default: \code{ntop = NULL})
+#'   if all features should be used. (Default: \code{NULL})
 #'
-#' @param subset.row Vector specifying the subset of features to use for
+#' @param subset.row \code{Character Vector}. Specifies the subset of features to use for
```

Comment:
In fact, this can be \code{Character vector}, \code{integer vector} or \code{NULL}. `

--> check other function also that they list all possible classes.

Reply:
should we consider NULL as a class?